Source/Core/Core/AchievementManager.cpp

```diff
@@ -401,17 +402,20 @@ void AchievementManager::FilterApprovedPatches(std::vector<PatchEngine::Patch>&
     if (!verified)
     {
       patch_itr = patches.erase(patch_itr);
-      OSD::AddMessage(
-          fmt::format("Failed to verify patch {} from file {}.", patch_itr->name, game_ini_id),
-          OSD::Duration::VERY_LONG, OSD::Color::RED);
-      OSD::AddMessage("Disable hardcore mode to enable this patch.", OSD::Duration::VERY_LONG,
-                      OSD::Color::RED);
+      any_patches_denied = true;
     }
     else
     {
       patch_itr++;
     }
   }
+  if (any_patches_denied)
+  {
+    OSD::AddMessage("Unverified patches found and disabled.", OSD::Duration::VERY_LONG,
+                    OSD::Color::RED);
+    OSD::AddMessage("Disable hardcore mode to enable unapproved patches.", OSD::Duration::VERY_LONG,
+                    OSD::Color::RED);
```

Comment:
Please don't mix terminology like this. Are they approved or are they verified, which is it?

Reply:
Also I just tried this and it works, you can use a newline in the message and ImGUI will respect that. So you don't need to do two separate messages here.